docs/integrations-guides/rollup-guides/orbit/overview.md

```diff
@@ -34,6 +36,42 @@ For replicating EigenDA interactions in a local testing environment, we utilize
 
 [EigenDA Proxy](https://github.com/Layr-Labs/eigenda-proxy) is used for secure and optimized communication between the rollup and the EigenDA disperser. Arbitrum uses the [*Simple Commitment Mode*](https://github.com/Layr-Labs/eigenda-proxy?tab=readme-ov-file#simple-commitment-mode) for client/server interaction and representing DA certificates. Read more about EigenDA Proxy and its respective security features [here](./../../dispersal/clients/eigenda-proxy.md).
 
+### Posting batches to EigenDA
+
+Please ensure that changes made to batch poster configs are globally applied across all your batch poster instances. If not, there could be inconsistencies that arise due to deviations in collective processing logic. To learn more about the Arbitrum batch poster please advise our following overview [spec](https://hackmd.io/@epociask/ByHk6x_TC).
+
+**Adjusting maximum batch size**
+
+Currently, the batch poster defaults to a maximum of `16mib` when dispersing batches to EigenDA. This can be adjusted to a lower threshold directly within the batch poster section of your node config:
+
+```
+    "node": {
+        ...
+        "batch-poster": {
+            "enable": true,
+            ...
+            "max-eigenda-batch-size": 12_000_000, // 12 MB
+        }
+    }
+```
+
+
+**Enabling Failover**
+
+To remove a trust assumption on the liveness of EigenDA for the liveness of the rollup, we've extended the Arbitrum Nitro batch poster's logic to support opt-in failover to other DA destinations (e.g, AnyTrust, 4844, calldata) in the event of indicated service unavailability from EigenDA. This logic is disabled by default but can be enabled by making the following update to your batch poster config with the following field:
+```
+    "node": {
+        ...
+        "batch-poster": {
+            "enable": true,
+            ...
+            "enable-eigenda-failover": true, 
+        }
+    }
+```
+
+To learn more about our Arbitrum failover design methodology, please advise the following [spec](https://hackmd.io/@epociask/SJUyIZlZkx).
+
 # Diff Overview 
```

Comment:
What's the diff specifically referencing? Is this our forked Arbitrum Orbit Nitro Fork? 

If so can we be explicit?

Reply:
the below diff demarcates between between each specific fork